examples/run/run.cpp

```diff
@@ -702,6 +803,27 @@ class LlamaData {
 
         return sampler;
     }
+
+    std::string initialize_chat_template(const llama_model_ptr & model, const Opt & opt) {
+        // if no template file doesn't exists, just return an empty string
+        struct stat info;
+        if (stat(opt.chat_template_.c_str(), &info) != 0) {
+            return common_get_builtin_chat_template(model.get());
+        }
+
+        std::ifstream tmpl_file;
```

Comment:
Would prefer ggml_fopen over streams, but not necessary, most important is to get it working first :)